fasthttpadaptor/request.go

```diff
@@ -57,3 +58,7 @@ func ConvertRequest(ctx *fasthttp.RequestCtx, r *http.Request, forServer bool) e
 
 	return nil
 }
+
+func b2s(b []byte) string {
```

Comment:
A comment needs to be added here for gosec check to pass https://github.com/valyala/fasthttp/blob/v1.40.0/bytesconv.go#L326

Can copy from https://github.com/valyala/fasthttp/blob/v1.40.0/bytesconv.go#L320-L328

Reply:
Thank you, added it 👍